include/common_structs.h

```diff
@@ -127,9 +127,9 @@ typedef struct CamPosSettings {
 } CamPosSettings; // size = 0x1C
 
 typedef struct DmaTable {
-    /* 0x00 */ s32 start;
-    /* 0x04 */ s32 end;
-    /* 0x08 */ s32 dest;
+    /* 0x00 */ u8* start;
+    /* 0x04 */ u8* end;
+    /* 0x08 */ u8* dest;
```

Comment:
Addr for these three?

Reply:
Addr is u8[] so it unfortunately can't be used here (syntax errors)